samples/model-builder/model_registry/create_model_registry_sample.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_model_registry_create_model_registry_sample]
+
+from google.cloud import aiplatform
+
+
+def create_model_registry_sample(model_id: str, project: str, location: str):
+    """
+    Create a model registry.
+    Args:
+        model_id: The ID of the model.
+        project: The project name.
+        location: The location name.
+    Returns:
+        The created model registry.
+    """
+
+    # Initialize the client.
+    aiplatform.init(project=project, location=location)
+
+    # Initialize the model registry with the ID 'model_id'.The parent_name of create method can be also
+    # 'projects/<your-project-id>/locations/<your-region>/models/<your-model-id>'
+
+    model_registry = aiplatform.models.ModelRegistry(model=model_id)
```

Comment:
what's the difference between this and samples/model-builder/model_registry/create_default_model_sample.py?

Reply:
Hi @deanchen-google,

the `create_default_model_sample` initialises a Model Resource to represent an existing model version with alias 'default'. The `create_model_registry_sample` creates the model registry (all model versions) associated to a particular model_id

Reply:
But the input and output from 2 samples are all the same?